### DISCUSSION TITLE
Hola. El android Card no funciona en mi Volkswagen Tiguan 2020? Alguien puede ayudarme

### DISCUSSION THREAD

**Discussion by DavidFiestas** [community/community#126768] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

Necesito ayudar. Compre un andriod cart de github y no me funciona en mi tiguan 2020. gracias por la ayuda

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by ebndev** [comment]
Hi @DavidFiestas, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
